GVFS/GVFS.Installer.Mac/scripts/Distribution_GVFS.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<installer-gui-script minSpecVersion="1">
+    <pkg-ref id="com.vfsforgit.pkg"/>
+    <options customize="never" allow-external-scripts="true"/>
+    <choices-outline>
+        <line choice="default">
+            <line choice="com.vfsforgit.pkg"/>
+        </line>
+    </choices-outline>
+    <choice id="default"/>
+    <choice id="com.vfsforgit.pkg" visible="false">
+        <pkg-ref id="com.vfsforgit.pkg"/>
+    </choice>
+    <pkg-ref id="com.vfsforgit.pkg" version="1.2.3" onConclusion="none">VFSFORGITPACKAGEFILENAME</pkg-ref>
+    <installation-check script="InstallationCheck()"/>
+    <script>
+        function InstallationCheck(prefix) 
+        {
+            var blockingProcesses = GetBlockingProcesses();
+            if (blockingProcesses.length > 0)
+            {
+                my.result.message = "Quit these processes and try again - " + blockingProcesses.join(", ") + ".";
+                my.result.type = 'Fatal';
+                return false;
+            }
+            
+            my.result.message = "No blocking processes running.";
+            my.result.type = "Info";
+            return true;
+        }
+        
+        function GetBlockingProcesses()
+        {
+            var watchList = [ "Calculator", "TextEdit", "Preview" ];
```

Comment:
Will these be updated to the VFS4G/Git process names we care about?

Reply:
absolutely. updated to `"gvfs", "gvfs.mount", "git", "gitk"`